### PR TITLE
OCM-6285 | feat: Day1 SG support for HCP

### PR DESCRIPTION
**Pull request by hunterkepley**
**What this PR does / why we need it**:
Allows HCP clusters to have additional compute SGs on creation, following the introduced support for day 1 HCP SGs in CS/ROSA

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (OCM-xxxx)*:
Fixes #[OCM-6285](https://issues.redhat.com//browse/OCM-6285)

**Change type**
- [X] New feature
- [ ] Bug fix
- [ ] Build
- [ ] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [X] Subsystem tests

**Checklist**
- [X] Subject and description added to both, commit and PR.
- [X] Relevant issues have been referenced.
https://issues.redhat.com/browse/OCM-6285

### CODE REVIEW

File: provider/clusterrosa/hcp/datasource.go [hunk 1]
```diff
@@ -260,6 +260,11 @@ func (r *ClusterRosaHcpDatasource) Schema(ctx context.Context, req datasource.Sc
 				Description: deprecatedMessage,
 				Computed:    true,
 			},
+			"aws_additional_compute_security_group_ids": schema.ListAttribute{
+				Description: "AWS additional compute security group ids. " + common.ValueCannotBeChangedStringDescription,
+				ElementType: types.StringType,
+				Computed:    true,
```
Comment (gdbranco) [inline provider/clusterrosa/hcp/datasource.go:266]: Why is it computed?

File: tests/tf-manifests/rhcs/clusters/rosa-hcp/output.tf [hunk 2]
```diff
@@ -17,3 +17,7 @@ output "properties" {
 output "tags" {
   value = rhcs_cluster_rosa_hcp.rosa_hcp_cluster.tags
 }
+
+output "additional_control_plane_security_groups" {
```
Comment (gdbranco) [inline tests/tf-manifests/rhcs/clusters/rosa-hcp/output.tf:21]:
```suggestion
output "additional_compute_security_groups" {
```

File: tests/tf-manifests/rhcs/clusters/rosa-hcp/main.tf [hunk 3]
```diff
@@ -63,6 +63,7 @@ resource "rhcs_cluster_rosa_hcp" "rosa_hcp_cluster" {
   proxy                        = var.proxy
   aws_subnet_ids               = var.aws_subnet_ids
   private                      = var.private
+  aws_additional_compute_security_group_ids       = var.additional_compute_security_groups
```
Comment (gdbranco) [inline tests/tf-manifests/rhcs/clusters/rosa-hcp/main.tf:66]: Could you adjust indentation?

**Review by robpblake — APPROVED** [review]
/lgtm

**Pull request merged**